src/sections/quiz/quiz-header.js

```diff
@@ -0,0 +1,36 @@
+import Button from "@mui/material/Button";
+import {SvgIcon} from "@mui/material";
+import Container from "@mui/material/Container";
+import Box from "@mui/material/Box";
+import Logo from "../../components/svg-color/Logo.svg";
+import {Lora} from '@next/font/google';
+
+const lora = Lora({preload: false});
+
+export default function QuizHeader() {
+  return (
+    <Box
+      component="header"
+      sx={{
+        backgroundColor: '#f9f4ee',
```

Comment:
Customize palette and refer to value from palette here.

Reply:
backgroundColor: theme.palette.background.quiz,